app/mailers/article_mailer.rb

```diff
@@ -9,13 +9,13 @@
 class ArticleMailer < MandrillMailer::TemplateMailer
   include ActionView::Helpers::UrlHelper
 
-  default from: 'orientation@codeschool.com'
+  default from: ENV['DEFAULT_FROM_EMAIL'] || 'orientation@codeschool.com'
 
   def notify_author_of_staleness(articles)
     author = articles.last.author
     mandrill_mail template: 'stale-article-alert',
-                  subject: 'Some of your Orientation articles might be stale',
-                  from_name: 'Orientation',
+                  subject: 'Some of your articles might be stale',
```

Comment:
You'd need to update the article_mailer_spec for this change. Although I sort of object to the removal of "Orientation" here. This strips the subject line of important context .